src/element.js

```diff
@@ -128,11 +128,11 @@ const Props = {
   },
   set mount(v) {
     if (typeof v === 'object' || (isObjectString(v) && (v = parseToObject(v)))) {
-      if (v.hasOwnProperty('x')) {
+      if (Object.prototype.hasOwnProperty.call(v, 'x')) {
```

Comment:
Maybe we should use `in` instead `hasOwnProperty`. I think it's marginally faster, but it's definitely more readable

Reply:
Agreed @michielvandergeest .  Replaced _hasOwnProperty_ check with _in_